src/components/Search/CompareOverTime.tsx

```diff
@@ -15,7 +15,7 @@ import { Changeset, Repository } from '../../types/state';
 import { Framework, TimeRange } from '../../types/types';
 import CancelAndCompareButtons from './CancelAndCompareButtons';
 import EditButton from './EditButton';
-import FrameworkDropdown from './FrameworkDropdown';
+import SearchFrameworkComponent from './SearchFrameworkComp';
```

Comment:
nit: use SearchFrameworkDropdown, so that it's clearer what this is when looking at the code here.